tests/plugins/test_koji_import.py

```diff
@@ -1152,6 +1152,9 @@ def test_koji_import_success(self, tmpdir, apis, docker_registry,
         workflow.prebuild_results[CheckAndSetRebuildPlugin.key] = is_autorebuild
         if pulp_pull:
             workflow.exit_results[PLUGIN_PULP_PULL_KEY] = pulp_pull
+
+        workflow.builder.image_id = expect_id
```

Comment:
The test should verify that `expect_id` is the same as `workflow.builder.image_id` in the end, no?

Reply:
test is verifying that metadata was set, verifying functionality of pulp pull is up to pulp_pull test
https://github.com/projectatomic/atomic-reactor/pull/840/files#diff-7a4c2d84966bd20f8d8c269e05f6df49R1184

Reply:
Okay, got it